File: DinoRunnerObjC-Demo/DinoRunnerObjC-Demo/ViewController.m

#import "ViewController.h"

#import <DinoRunner/DinoRunner-Swift.h>

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UIButton *playButton;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)play:(id)sender {
    [DinoRunner startGame];
}

@end

File: DinoRunner-SPM-Demo/DinoRunner-SPM-Demo/ViewController.swift

import UIKit
import DinoRunner

class ViewController: UIViewController {

    @IBOutlet weak var playButton: UIButton!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    @IBAction func play(_ sender: Any) {
        DinoRunner.startGame()
    }
}

